superset/reports/notifications/slack.py

```diff
@@ -193,6 +219,46 @@ def send(self) -> None:
                     )
             else:
                 client.chat_postMessage(channel=channel, text=body)
+
+    @backoff.on_exception(backoff.expo, SlackApiError, factor=10, base=2, max_tries=5)
+    @statsd_gauge("reports.slack.send")
+    def send(self) -> None:
+        global_logs_context = getattr(g, "logs_context", {}) or {}
+        try:
+            client = get_slack_client()
+            title = self._content.name
+            body = self._get_body()
+
+            try:
+                channels = self._get_channels(client)
+            except SlackApiError:
+                logger.warning(
+                    "Slack scope missing. Using deprecated API to get channels. Please update your Slack app to use the new API.",
```

Comment:
Do we know what scope is missing here? Can we add it to the error message?

Reply:
I assume it will be sent by slack to the error message.

Reply:
We expect `channels:read` as per the difference between v1 and v2 apis, but it could be anything if they're not set up correctly.